src/jqLite.js

```diff
@@ -56,7 +56,7 @@
  * - [`after()`](http://api.jquery.com/after/)
  * - [`append()`](http://api.jquery.com/append/)
  * - [`attr()`](http://api.jquery.com/attr/) - Does not support functions as parameters
- * - [`bind()`](http://api.jquery.com/bind/) - Does not support namespaces, selectors or eventData
+ * - (_deprecated_, use `on()`) [`bind()`](http://api.jquery.com/bind/) - Does not support namespaces, selectors or eventData
```

Comment:
That's how it looks like:
<img width="718" alt="screen shot 2016-10-05 at 12 42 29" src="https://cloud.githubusercontent.com/assets/1758366/19110250/3a20dc02-8af9-11e6-982c-b82f5dedd380.png">